static/src/components/fieldsconfig/categorical_field_config.vue

```diff
@@ -5,7 +5,7 @@
         :name="name"
         :type="type"
         :retry-for-consensus="retryForConsensus"
-        :editable="true"
+        :is-editable="true"
```

Comment:
I don't see this prop `is-editable` on this component `static/src/components/fieldsconfig/field_config_base.vue`